lib/types/luks.nix

```diff
@@ -143,19 +143,21 @@ in
 
             askPassword() {
               if [ -z ''${IN_DISKO_TEST+x} ]; then
-                set +x
                 promptSecret "Enter password for ${config.device}: " password
                 promptSecret "Enter password for ${config.device} again to be safe: " password_check
                 export password
                 [ "$password" = "$password_check" ]
```

Comment:
probably better to use exit 1 if the password don't match instead of setting the debug flags